debug_toolbar/panels/__init__.py

```diff
@@ -21,6 +24,10 @@ def panel_id(self):
 
     @property
     def enabled(self) -> bool:
+        # check if the panel is async compatible
+        if not self.is_async and isinstance(self.toolbar.request, ASGIRequest):
+            return False
```

Comment:
I think we're missing a test for this logic

Reply:
this should be covered by panels test seprately?